src/solver/simulation/solver.hxx

```diff
@@ -1547,13 +1549,12 @@ void ISimulation<Impl>::loopThroughYears(uint firstYear,
                 study.runtime->currentYear[numSpace] = y;
             }
 
-            // If the year has not to be rerun, we skip the computation of the year Note that, when
-            // we enter for the first time in the "for" loop, all years of the set have to be rerun
+            // If the year has not to be rerun, we skip the computation of the year. 
+            // Note that, when we enter for the first time in the "for" loop, all years of the set have to be rerun
             // (meaning : they must be run once). if(!set_it->yearFailed[y]) continue;
 
             qs.add(new yearJob<ImplementationType>(this,
                                                    y,
-                                                   yearsIndicesCopy,
```

Comment:
`yearIndicesCopy` is no longer required, do not declare.

Reply:
Done